src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -191,7 +190,15 @@ const connectQueryRules: QueryRulesConnector = (render, unmount = noop) => {
     let onHelperChange: (state: HelperState) => void;
 
     return {
-      init({ helper, state, instantSearchInstance }) {
+      init({
+        helper,
+        state,
+        instantSearchInstance,
+      }: {
+        helper: Helper;
+        state: HelperState;
+        instantSearchInstance: InstantSearch;
+      }) {
```

Comment:
TypeScript is not able to infer it from the `Widget`?

Reply:
Now that we made them required, no need for this!